framework/console/Request.php

```diff
@@ -73,7 +73,11 @@ public function resolve()
                 }
             } elseif (preg_match('/^-(\w+)(?:=(.*))?$/', $param, $matches)) {
                 $name = $matches[1];
-                $params['_aliases'][$name] = isset($matches[2]) ? $matches[2] : true;
+                if (is_numeric($name) === false) {
```

Comment:
is_numeric already returns a boolean http://php.net/manual/en/function.is-numeric.php. No need for a comparison here